src/index.html

```diff
@@ -1,50 +1,70 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" type="text/css" href="css/style.css">
-    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
-    <title>freeCodeCamp Manila</title>
-  </head>
-  <body>
-    <section class="gallery">
-      <h2 class="gallery-title">Our Meetups Gallery</h2>
-
-      <div class="gallery-grid-container">
-        <div class="card gallery-grid-leftCol">
-          <img class="card-photo" src="images/meetup-august.jpg" alt="meetup-august">
-          <div class="card-text-container">
-            <p class="card-title">August RoundTable Meetup</p>
-            <p class="card-text card-secondary-text text-supporting"><img class="map-icon" src="images/map-pin.svg"> Bo's Coffee, Ayala</p>
-          </div>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+  <title>freeCodeCamp Manila</title>
+</head>
+
+<body>
+
+  <section class="community">
+    <h2 class="community-title">Join the Community</h2>
+    <p class="community-text">
+      Engage in a friendly community of campers on our Facebook Group. Anyone Interested in learning
+      how to code is welcome, regardless of your current level of knowledge. Attend our monthly
+      meetups and stay accountable, get help, and learn together.
+    </p>
+    <div class="community-btn">
+      <a href="https://facebook.com/groups/free.code.camp.manila/" target="blank">Join our Facebook Group</a>
```

Comment:
Please change `target="blank"` to target="_blank". Also add `rel="noopener noreferrer"` [here's why?](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)